src/client/driver/index.js

```diff
@@ -35,7 +35,7 @@ export default class ClientDriver {
     }
 
     start () {
-        browser.startHeartbeat(this.heartbeatUrl, hammerhead.nativeMethods.XMLHttpRequest);
+        browser.startHeartbeat(this.heartbeatUrl, hammerhead.createNativeXHR);
```

Comment:
you've changed a class constructor to a creator (function). So, I suppose you should update usage of it [here](https://github.com/DevExpress/testcafe/blob/b0863433f0b5dd11815f8b185130d9789e172ded/src/client/browser/index.js#L37) and [here](https://github.com/DevExpress/testcafe/blob/b0863433f0b5dd11815f8b185130d9789e172ded/src/client/browser/index.js#L43)

It should like
```js
export function checkStatus (statusUrl, createXHR){...}

function sendXHR (url, createHXR) {
    return new Promise((resolve, reject) => {
        var xhr = createHXR();
...
```